app/views/checkout/payment.scala.html

```diff
@@ -14,168 +12,95 @@
-                                @fragments.forms.errorMessage("This field is required")
-                            </div>
-                            <div class="form-field js-checkout-street">
-                                <label class="label optional-marker" for="address-line-2">Address line 2</label>
-                                <input type="text" class="input-text js-input" id="address-line-2" name="personal.address.address2" value="@form.data.get("personal.address.address2")">
-                                @fragments.forms.errorMessage("This field is required")
-                            </div>
-                            <div class="form-field js-checkout-town">
-                                <label class="label" for="address-town">Town/City</label>
-                                <input type="text" class="input-text js-input" id="address-town" name="personal.address.town" value="@form.data.get("personal.address.town")">
-                                @fragments.forms.errorMessage("This field is required")
+                        <div class="field-panel__intro">
+                            <div class="field-note field-note--offset">
+                                @if(userIsSignedIn) {
+                                    <span class="field-note__label">Not you?</span>
+                                    <a href="@idWebAppSignOutUrl(renderCheckout())">Sign out</a>
+                                } else {
+                                    <span class="field-note__label">Already have a Guardian account?</span>
+                                    <a href="@idWebAppSigninUrl(renderCheckout())">Sign in</a>
+                                }
                             </div>
                         </div>
-                        <div class="form-field js-checkout-postcode">
-                            <label class="label" for="address-postcode">Postcode</label>
-                            <input type="text" class="input-text js-input input-text input-text--small" id="address-postcode" name="personal.address.postcode" value="@form.data.get("personal.address.postcode")">
-                            @fragments.forms.errorMessage("Please enter a valid postal code")
+                        <div class="field-panel__edit">
+                            <a href="#yourDetails" class="text-button u-button-reset js-edit-your-details">Edit</a>
                         </div>
+                        <div class="field-panel__fields">
+                            @fragments.checkout.fieldsPersonal(form, userIsSignedIn)
 
-                        <div class="actions">
-                            <a href="#paymentDetails" class="button button--primary button--large js-checkout-your-details-submit">Continue</a>
+                            <div class="actions">
+                                <a href="#paymentDetails" class="button button--primary button--large js-checkout-your-details-submit">Continue</a>
```

Comment:
Yep, the `id` is still on the container below, l63 `<div id="paymentDetails" class="field-panel is-collapsed js-fieldset-payment-details">`